pkg/workflow/json_schemas/data_test_summary.go

```diff
@@ -0,0 +1,51 @@
+package workflow
+
+const TestSummarySchema = `{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "Test Results",
+	"description": "Schema for security test results with severity and counts",
+	"type": "object",
+	"required": ["results", "type"],
+	"properties": {
+	  "results": {
+		"type": "array",
+		"minItems": 1,
+		"items": {
+		  "type": "object",
+		  "required": ["severity", "total", "open", "ignored"],
+		  "properties": {
+			"severity": {
+			  "type": "string",
+			  "enum": ["high", "medium", "low", "critical"]
```

Comment:
Question: Does it have to be an enum?

Reply:
I mean do we actually care about the valiue?

Reply:
I think you're right in this instance it doesn't actually matter as we have nothing looking at the severity value yet. My preference is to use this schema to document what we _expect_ to get here. We can always relax to a string if needed, but it would be harder to move in the other direction.